threaddb/mail.go

```diff
@@ -43,40 +51,68 @@ type Message struct {
 	ReadAt    int64  `json:"read_at"`
 }
 
+// SentboxMessage represents the sentbox threaddb collection schema.
+type SentboxMessage struct {
```

Comment:
Only diff here is that there's no `ReadAt`, but I'd imagine these could diverge more in the future, so opted to go the more verbose path and have two. The client-side still just uses a single model.